tests/test_classifier/test_confusion_matrix.py

```diff
@@ -190,6 +190,7 @@ def test_percent_mode(self):
             ),
         )
 
+    @pytest.mark.xfail(reason="class filtering is not currently supported")
```

Comment:
ConfusionMatrix was the only classifier visualizer that previously supported class-based filtering.